### ISSUE TITLE
Any plans to take the documentation out of "beta"?

### ISSUE THREAD

**Issue by almostagile** [knex/knex#5960]
I noticed in a Slack preview of a shared documentation link that it was described as "Beta knex.js documentation". I think that comes from the following line.

https://github.com/knex/documentation/blob/68a42405f756034821f7af7e58016105d9eafa79/.vitepress/config.ts#L6

At what point do you expect the documentation to come out of beta?

**Comment by rubengmurray** [comment]
This can be moved to knex/documentation repo I would've thought @mercmobily

**Comment by mercmobily** [comment]
No, the documentation repo is legacy and must be archived. I made the mistake of adding an issue there, and will need to move it over here.

**Comment by mercmobily** [comment]
Thank you for raising this question about the documentation status. You're absolutely right - the "Beta knex.js documentation" description in the VitePress configuration has been there since the documentation was migrated from the legacy documentation repository to the main knex repository in March 2024 (commit a58208cd).

Looking at the current state of the documentation:

The documentation site at knexjs.org is fully functional and actively maintained. It includes comprehensive guides, API references, migration documentation, and FAQ sections. The VitePress integration (completed in 2022) provides a modern documentation experience with search capabilities via Algolia and proper versioning.

**Current situation**

The "beta" label in line 6 of `/docs/.vitepress/config.ts` appears to be a remnant from the initial VitePress migration that was never updated. The documentation itself is stable and production-ready - it's just this metadata description that hasn't been updated.

The fix is straightforward - update the description in the VitePress configuration to remove the "beta" designation. This would involve changing:

```typescript
description: 'Beta knex.js documentation.',
```

to something more appropriate like:

```typescript
description: 'Knex.js - SQL query builder for Node.js',
```

or

```typescript
description: 'Official documentation for Knex.js SQL query builder',
```

This aligns with how other established ORMs and query builders describe their documentation. For comparison, Prisma uses "Next-generation ORM for Node.js & TypeScript", whilst Sequelize uses "A modern TypeScript and Node.js ORM".

The documentation is mature and should reflect that in its metadata.
